src/nfv_test_api/v2/controllers/ue_5g.py

```diff
@@ -23,10 +23,10 @@
 from nfv_test_api.host import Host
 from nfv_test_api.v2.controllers.common import add_model_schema
 from nfv_test_api.v2.data.common import InputSafeSupi
-from nfv_test_api.v2.data.ue import UE, UECreate, UEStatus
-from nfv_test_api.v2.services.ue import UEService, UEServiceHandler
+from nfv_test_api.v2.data.ue_5g import UE, UECreate, UEStatus
+from nfv_test_api.v2.services.ue_5g import UEService, UEServiceHandler
 
-namespace = Namespace(name="ue", description="Basic user equipment management")
+namespace = Namespace(name="ue_5g", description="Basic 5G user equipment management")
```

Comment:
This will change the url, right? This means it will break all our tests.  How do you plan on updating the test suite?

Reply:
I think I can already make a branch in mpn to fix it and merge it as soon as I rebuild the labs with the new image.